lte/gateway/c/core/oai/tasks/s1ap/s1ap_mme.h

```diff
@@ -82,4 +82,12 @@ void s1ap_remove_ue(s1ap_state_t* state, ue_description_t* ue_ref);
  **/
 void s1ap_remove_enb(s1ap_state_t* state, enb_description_t* enb_ref);
 
+/**
+ * \brief Callback function to remove target ue if matches sctp assoc id
+ * @return false if UE not found, true otherwise
+ */
```

Comment:
is this comment correct?

Reply:
it is correct, a better name for this function could be `s1ap_remove` instead of dump, I will update